desktop/components/follow_button/view.coffee

```diff
@@ -51,7 +51,7 @@ module.exports = class FollowButton extends Backbone.View
 
     unless @following
       mediator.trigger 'clickFollowButton'
-      return if NEW_ARTIST_PAGE_CTA is 'new_cta' and ON_ARTIST_PAGE
+      return if ARTIST_PAGE_CTA_ENABLED
```

Comment:
In this case, we want to early return from the follow button, and instead trigger the new CTA takeover.